File: CUTEProject/bundles/ch.hsr.ifs.testframework/src/ch/hsr/ifs/testframework/model/Messages.java
package ch.hsr.ifs.testframework.model;

import org.eclipse.osgi.util.NLS;


/**
 * @since 3.0
 */
public class Messages extends NLS {

   private static final String BUNDLE_NAME = "ch.hsr.ifs.testframework.model.messages";
   public static String        ModellBuilder_0;
   static {
      NLS.initializeMessages(BUNDLE_NAME, Messages.class);
   }

   private Messages() {}
}

File: CUTEProject/bundles/ch.hsr.ifs.testframework/src/ch/hsr/ifs/testframework/model/Messages.kt
package ch.hsr.ifs.testframework.model

import org.eclipse.osgi.util.NLS
import kotlin.jvm.JvmField


/**
 * @since 3.0
 */
public class Messages : NLS() {

   companion object {
      private const val BUNDLE_NAME = "ch.hsr.ifs.testframework.model.messages"

	   @JvmStatic
      lateinit var ModellBuilder_0: String

      init {
         NLS.initializeMessages(BUNDLE_NAME, Messages::class.java)
      }
   }

}
